benchmarks/BayesianInference/DiffEqBayesLotkaVolterra.jmd

```diff
@@ -67,6 +74,35 @@ The solution converges for tolerance values lower than 1e-3, lower tolerance lea
 @btime bayesian_result_stan = stan_inference(prob,t,data,priors,num_samples=10_000,print_summary=false, vars = (DiffEqBayes.StanODEData(), InverseGamma(2, 3)))
 ```
 
+### Direct Turing.jl
+
+```julia
+@model function fitlv(data, prob)
+    # Prior distributions.
+    σ ~ InverseGamma(2, 3)
+    α ~ truncated(Normal(1.5, 0.5), 0.5, 2.5)
+    β ~ truncated(Normal(1.2, 0.5), 0, 2)
+    γ ~ truncated(Normal(3.0, 0.5), 1, 4)
+    δ ~ truncated(Normal(1.0, 0.5), 0, 2)
+
+    # Simulate Lotka-Volterra model. 
+    p = SA[α, β, γ, δ]
+    _prob = remake(sprob, p = p)
+    predicted = solve(_prob, Tsit5(); saveat=0.1)
```

Comment:
There seems to be a typo (`sprob` instead of the argument `prob` - I assume one rather wants to pass `sprob` below instead of hardcoding it here). However, maybe it would be even simpler to use
```suggestion
    predicted = solve(prob, Tsit5(); p=p, saveat=0.1)
```
Or are there any advantages of an explicit `remake`?

Reply:
Type stability and inference improvements.

Reply:
Ah interesting. A bit unfortunate since using `remake` looks a bit less intuitive/straightforward.

Then I'll change my suggestion to
```suggestion
    _prob = remake(prob, p = p)
    predicted = solve(_prob, Tsit5(); saveat=0.1)
```